Consolidate pros/cons sections in Coin Selection Page (#436)
BitcoinDesign/Guide: guide/payments/coin-selection.md

#### Speed

> Higher fees will increase the likelihood of your transaction being prioritized by miners and included in the next block, thereby increasing the speed of your transaction.

Reviewer: ```suggestion
Higher fees will increase the likelihood of your transaction being prioritized by miners and included in the next block, thereby increasing the speed of your transaction being confirmed.
```

Small bit but It's important to add the context that when talking about speeding up a transaction it pertains to it being confirmed in the network.

%}

{% include fact/pros.html %}
- Low friction
- Optimizes for cost, speed, or privacy
- Minimizes change outputs and decreases balance exposure
{% include fact/close.html %}

{% include fact/cons.html %}
- Doesn't optimize for all three features (speed, cost, privacy)
- Can select UTXOs associated with targeted dusting attacks, resulting in privacy loss
- UTXOs selected from unrecognized clusters can expose private data of wallet addresses, balances, and contact payment information.
- Reducing the inputs (byte size) of bitcoin transactions doesn’t usually optimise for minimal change outputs, exposing larger amounts of sensitive wallet data.

> - Freezing certain coins reduces the size of a wallet's UTXO pool, and may hinder an automatic algorithm’s ability to optimise for speed, cost, or privacy.

Reviewer 2: ```suggestion
- Freezing coins reduces the available UTXOs an automatic coin selection algorithm can choose from so the resulting transaction may not be properly optimized.
```

Reviewer 2: can be less wordy...

%}

{% include fact/pros.html %}
- Low friction
- Optimizes for cost, speed, or privacy
- Minimizes change outputs and decreases balance exposure
{% include fact/close.html %}

{% include fact/cons.html %}
- Doesn't optimize for all three features (speed, cost, privacy)
- Can select UTXOs associated with targeted dusting attacks, resulting in privacy loss
- UTXOs selected from unrecognized clusters can expose private data of wallet addresses, balances, and contact payment information.
- Reducing the inputs doesn’t usually optimise for minimal change outputs, exposing larger amounts of sensitive wallet data.

> - Freezing coins reduces the available UTXOs an automatic coin selection algorithm can choose from so the resulting transaction may not be properly optimized.

Reviewer 2: ```suggestion
- Freezing UTXOs prevents coin selection algorithms from using them, so the resulting transaction may not be properly optimized.
```
